yt/visualization/volume_rendering/transfer_functions.py

```diff
@@ -603,7 +603,15 @@ def y_format(y, pos):
         ax.set_xlabel("Value")
 
     def vert_cbar(
-        self, resolution, log_scale, ax, label=None, label_fmt=None, *, size=10
+        self,
+        resolution,
+        log_scale,
+        ax,
+        *,
+        label=None,
+        label_fmt=None,
+        label_fontsize=10,
+        size=10,
```

Comment:
`label` and `label_fmt` are currently allowed-positional, so making them suddenly keyword-only is a breaking change
```suggestion
        self,
        resolution,
        log_scale,
        ax,
        label=None,
        label_fmt=None,
        *,
        label_fontsize=10,
        size=10,
```

Reply:
updated